pkg/endpoint/policy.go

```diff
@@ -67,6 +67,23 @@ var (
 	}
 )
 
+// RegenerationContext provides context to regenerate() calls to determine
```

Comment:
Can be done, I wasn't sure what the threshold should be for putting things in their own file since there isn't really any code associated directly with the `RegenerationContext`.